### ISSUE TITLE
fish shell -- input freeze after some typing

### ISSUE THREAD

**Issue by rmsthebest** [ish-app/ish#1764]
Iphone 12, IOS 15.3

how to reproduce:
```sh
apk update
apk add fish
apk add libuser
touch /etc/login.defs
mkdir /etc/default
touch /etc/default/useradd

passwd
> <password>
> <password>

#Change current USER's shell:
lchsh root
> <password>
> /usr/bin/fish
```
reboot, then type a bit and it will get stuck very quickly, especially after using backspace

**Comment by PeachIceTea** [comment]
Same issue, write a few characters and it hangs. Little video of it happening: https://i.imgur.com/bvRPbtf.mp4

iPhone 13 mini, iOS 15.4

**Comment by jeffok** [comment]
I have the same problem

**Comment by tekacs** [comment]
One more report of this from me — hopefully this can have its unconfirmed tag struck given the video above and multiple reports.

**Comment by lukasbestle** [comment]
Having the same issue.

**Comment by Mnpn** [comment]
Hi! We've recently released iSH 1.3 which seems to resolve this issue (fish appears to be running well for me). Please let us know if you're still running into this problem, and feel free to re-open this issue if so! Thank you!

**Comment by iovis** [comment]
it seems to working on my end!
